src/toil/fileStore/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2015-2019 Regents of the University of California
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+
+__all__ = ['fileStore', 'nonCachingFileStore', 'cachingFileStore']
+from toil.fileStore.fileStore import *
```

Comment:
I'm a bit leery of using `*`, especially since importing all of these names into one file seems to be inviting collision.

Would you be amenable to something like:
```
import toil.fileStore.fileStore as fileStore
import toil.fileStore.nonCachingFileStore as nonCachingFileStore
import toil.fileStore.cachingFileStore as cachingFileStore
```